README.md

```diff
@@ -18,6 +18,12 @@ We deploy using **GitHub Actions** for CI/CD.
 
 You will need `npm` and Node installed, and also Java (for `serverless-dynamodb-local`)
 
+### Software Requirement
+- [Node 14](https://nodejs.org/en/download/)
+- [Java 16 JDK](https://www.oracle.com/java/technologies/javase-jdk16-downloads.html)
+- [Python 3.9](https://www.python.org/downloads/)
```

Comment:
What did you wind up needing Python for?

Reply:
It was one of the requirements for node-gyp. https://github.com/nodejs/node-gyp

Reply:
gotcha, so also a windows-only requirement?

Reply:
Possibly? Their readme says its needed for Mac and Linux as well